app/controllers/moderation/meetings_controller.rb

```diff
@@ -38,7 +40,9 @@ def index
               row.push meeting.title
               row.push meeting.address
               row.push meeting.attendee_count
+              row.push meeting.interventions
               row.push meeting.organization_count
+              row.push meeting.proposals.count
```

Comment:
I don't think this is gonna scale :S. We should probably cache this or something...